pinot-controller/src/main/java/org/apache/pinot/controller/helix/ControllerRequestClient.java

```diff
@@ -169,6 +170,39 @@ public void deleteSegments(String tableName, TableType tableType)
     }
   }
 
+  public PinotLLCRealtimeSegmentManager.PauseStatus pauseConsumption(String tableName)
+      throws IOException {
+    try {
+      SimpleHttpResponse response = HttpClient.wrapAndThrowHttpException(_httpClient.sendJsonPostRequest(new URL(
+          _controllerRequestURLBuilder.forPauseConsumption(tableName)).toURI(), null));
+      return JsonUtils.stringToObject(response.getResponse(), PinotLLCRealtimeSegmentManager.PauseStatus.class);
```

Comment:
We don't want `ControllerRequestClient` to depend on `PinotLLCRealtimeSegmentManager` right? Should we move `PauseStatus` to the controller/api package? We may rename or move the PinotLLCRealtmeSegmentManager at some point, and we don't want to rebuild clients just because. Perhaps there are other such uses in this file, maybe we should pay attention and fix those as well (on a separate PR/issue, of course) but we can stop introducing new ones

Reply:
Moved to api module.